build.gradle

```diff
@@ -175,7 +175,7 @@ task gems(type: Exec) {
     workingDir 'src/dist/gems'
     executable 'sh'
     // HACK: Workaround for bundler/bundler#4494
```

Comment:
If this version fixes the Bundler issue you can remove all instances where the workaround was used along with the comments. I believe previously we did not specify the version explicitly, opting to just use the latest in all cases.